doc/airline.txt

```diff
@@ -847,6 +847,9 @@ localsearch <https://github.com/mox-mox/vim-localsearch>
 * enable/disable localsearch indicator integration >
   let g:airline#extensions#localsearch#enabled = 1
 
+* show only when localsearch is off (shows 'GS' denoting GlobalSearch)  >
+  let g:airline#extensions#localsearch#enabled = 0
```

Comment:
copy paste error?

Reply:
The option should show the value for which the text is true? Okay.... I thought it should show the default value :sweat_smile:

Fixed